crates/re_data_ui/src/item.rs

```diff
@@ -1,26 +0,0 @@
-use re_viewer_context::{Item, UiVerbosity, ViewerContext};
-
-use crate::DataUi;
-
-impl DataUi for Item {
```

Comment:
do we know what this was used for?

Reply:
Selection panel and time panel; but not anymore